extensions/nicklist-3.3.md

```diff
@@ -0,0 +1,26 @@
+---
+title: IRCv3 nicklist extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Evan Magaliff"
+    period: "2017"
+    email: "evan@muffinmedic.net"
+---
+## Description
+This document describes the format of the `nicklist` capability extension. 
+
+This specification allows for sending a list of nicks the client would like to use to the server. The server then attempts to change the user's nick to each in the list until a successful nick change occurs or the list is exhausted.
+
+The given nicklist takes the format: `:foo!bar@example.com NICK :nick1 nick2 nick3`
```

Comment:
Theres no need to make it a <trailing> parameter. Just make it a series of <middle> parameters.